### PR TITLE
Currency: Added build script and build info file for jaf-api

### PR DESCRIPTION
**Pull request by muskaan2199**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: j/jaf-api/jaf-api_2.1.1_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+MAVEN_VERSION=3.6.3
+wget http://mirrors.estointernet.in/apache/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
+tar -C /usr/local/ -xzf apache-maven-$MAVEN_VERSION-bin.tar.gz
+mv /usr/local/apache-maven-$MAVEN_VERSION /usr/local/maven
+rm apache-maven-$MAVEN_VERSION-bin.tar.gz
+export M2_HOME=/usr/local/maven
+export PATH=$PATH:$M2_HOME/bin
+
+yum install -y java-11-openjdk java-11-openjdk-devel java-11-openjdk-headless
+ export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-11.0.18.0.10-2.el8_7.ppc64le
```
Comment (nikhil-kalbande) [inline j/jaf-api/jaf-api_2.1.1_ubi_8.5.sh:37]: @muskaan2199 We can set like `export JAVA_HOME=/usr/lib/jvm` and not specific version.
Please check

File: j/jaf-api/jaf-api_2.1.1_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+export PATH=$PATH:$JAVA_HOME/bin
+
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+cd api
+if ! mvn -B -V -U -C -Poss-release clean verify org.glassfish.copyright:glassfish-copyright-maven-plugin:check -Dgpg.skip=true -Dcopyright.ignoreyear=true; then
```
[the remark this answers is not in the document]
Comment (muskaan2199) [reply j/jaf-api/jaf-api_2.1.1_ubi_8.5.sh:44]: @nikhil-kalbande  as per our earlier discussion on teams there's only this command in the ci.yml file. It builds and tests the package

File: j/jaf-api/jaf-api_2.1.1_ubi_8.5.sh (hunk 1 above)
Comment (muskaan2199) [reply j/jaf-api/jaf-api_2.1.1_ubi_8.5.sh:37]: @nikhil-kalbande I tried. It gives me an error

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: j/jaf-api/build_info.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "package_name" : "jaf-api",
+    "github_url": "https://github.com/jakartaee/jaf-api",
+    "version": "2.1.1",
+    "default_branch": "master",
+    "package_dir": "j/jaf-api/",
+    "build_script": "jaf-api_2.1.1_ubi_8.5.sh",
+
```
[the remark this answers is not in the document]
Comment (muskaan2199) [reply j/jaf-api/build_info.json:9]: @nikhil-kalbande fixed this

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**